index.html

```diff
@@ -10,27 +10,68 @@
 </head>
 <body>
     <header class="header">
-            <nav class="nav-bar">
-                <img src="assets/logo.svg" id="logo" alt="Page logo">
-                <img src="assets/nav-burger.svg" class="nav-btn" alt="Button to navigate">
-            </nav>
+        <nav class="nav-bar">
+            <img src="assets/logo.svg" id="logo" alt="Page logo">
+            <img src="assets/nav-burger.svg" class="nav-btn" alt="Button to navigate">
+        </nav>
     </header>
     <main>
-        <div class="header-desktop">
-            <nav class="nav-desktop">
-                <ul>
-                    <li><a href="#">Portfolio</a></li>
-                    <li><a href="#">About</a></li>
-                    <li><a href="#">Contact</a></li>
-                </ul>
-            </nav>
+        <div class="mobile-menu-container">
+            <div class="header-desktop">
+                <div class="cross"></div>
+                <nav class="nav-desktop">
+                    <ul>
+                        <li><a href="#work_section">Portfolio</a></li>
+                        <li><a href="#about_section">About</a></li>
+                        <li><a href="#contact_section">Contact</a></li>
+                    </ul>
+                </nav>
+                <nav class="nav-mobile">
+                    <ul>
+                        <li><a href="#work_section">Portfolio</a></li>
+                        <li><a href="#about_section">About</a></li>
+                        <li><a href="#contact_section">Contact</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+        <div class="blur-bg hide">
+            <div class="pop-up">
+                <img src="assets/Disabled.png" class="close-pop-up" alt="close-button">
+                <img src="assets/portfolio-img.jpg" class="img-pop-up" alt="portfolio-image">
+                <img src="assets/cancel-icon.png" class="close-pop-up-mobile" alt="portfolio-image">
+                <img src="assets/portfolio-mobile.png" class="img-pop-up-mobile" alt="portfolio-image">
+                <span class="btn-container desktop-pop-up-btn">
+                        <button class="see-live-btn"><a>See live</a></button>
+                        <button class="see-source-btn"><a>See source</a></button>
+                </span>
+                <div class="pop-up-heading">
+                    <!-- <h3>Keeping track of hundreds of
+                        components</h3> -->
+                    <!-- <ul class="tech-stack">
+                        <li class="tag">Codekit</li>
+                        <li class="tag">Github</li>
+                        <li class="tag">JavaScript</li>
+                        <li class="tag">Bootstrap</li> 
+                        <li class="tag">Terminal</li> 
+                        <li class="tag">Codepen</li> 
+                    </ul> -->
```

Comment:
Kindly remove all commented lines of code from your project so as to have a clean code that is easily maintainable 😄